### DISCUSSION TITLE
Handling  404 route '/home/<random route> with module federation gives error with net::ERR_ABORTED 404 (Not Found)

### DISCUSSION THREAD

**Discussion by web4-var** [module-federation/module-federation-examples#2829] [Q&A]
Hi , 

I have a shell container and   a stand alone MFE 

From shell i have a route  set up  eg: 
`<Route
                  index
                  element={
                    <CustomRoute >
                      <ErrorBoundary key={'/'}>
                        <InHouseComponent />
                      </ErrorBoundary>
                    </CustomRoute>
                  }
                />
                <Route
                  path="food/*"
                  element={
                    <CustomRoute>
                      <ErrorBoundary key={'food'}>
                        <FoodComponent `/>`
                      </ErrorBoundary>
                    </CustomRoute>
                  }
                />
                <Route
                  path="/*"
                  element={
                    <ErrorHandling
                      errorMsg="Route not present"
                  />
                  }
                />
              </Routes>`

When the user goes to /home route error boundary is kicked in and i am able to see error page  but  anything after the route /home/ 

returns an error with https://localhost.com:8080/home/main.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
 https://localhost.com:8080/home/remoteEntry.js net::ERR_ABORTED 404 (Not Found)

Can some one  help why /home/ is creating a remoteEntry.js as such and not /home

**Comment by ScriptedAlchemy** [comment]
Use a runtime plugin or getPublicPath likely it's the base path messing with the auto mode the bundlers supply. Check out doc site for getPublicPath
